tests/dragonfly/pymemcached_test.py

```diff
@@ -154,12 +154,8 @@ def test_expiration(memcached_client: MCClient):
     assert memcached_client.get("key1") == b"value1"
     assert memcached_client.get("key2") == b"value2"
     assert memcached_client.get("key3") == b"value3"
-    time.sleep(1)
     assert memcached_client.set("key3", "value3", int(time.time()) - 200)
-    assert memcached_client.get("key1") == b"value1"
-    assert memcached_client.get("key2") == b"value2"
-    assert memcached_client.get("key3") == None
-    time.sleep(1)
+    time.sleep(2)
```

Comment:
Could you plz move `assert memcached_client.get("key3") == None` above this line/sleep? The reason is that the bug was that we did not delete previously existed elements when a set command got called with an expired TTL. Therefore, to adhere to this premise we must assert that the operation had an `immediate effect`. (Yes from the semantics and your PR we know that is the case since we call OpDel explicitly but it would be good to express this in this test)